.github/workflows/regression-report.yml

```diff
@@ -219,10 +221,13 @@ jobs:
       - name: Get message
         env:
           MSG: ${{ needs.make_report.outputs.message }}
+          DOWNLOAD_FAILED: ${{ needs.make_report.outputs.download_files_failed }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           mkdir -p .ci-temp
-          if [ -z "$MSG" ]; then
+          if [ "$DOWNLOAD_FAILED" == "true" ]; then
+            echo "Report generation failed. Please check the proper format: 'Please use 'GitHub, generate report for check_name/example_type'" > .ci-temp/message
```

Comment:
Let's add JOBS_LINK to let user quickly open execution and see all details 
See example how to do this below

Reply:
Done. Also, added new issue resolution. Updated the issue. https://github.com/checkstyle/checkstyle/issues/15394

Now, the workflow handles only active PRs.

Reply:
Please share link to your org PR with comments that have such links.

Reply:
> Please share link to your org PR with comments that have such links.

https://github.com/checkstyle-gsoc/checkstyle/pull/14

Reply:
![Screenshot_20240727-161221](https://github.com/user-attachments/assets/0a411213-ed32-46d0-91f3-96385b8f888b)